src/hardware/disney.h

```diff
@@ -45,8 +45,8 @@ class Disney final : public LptDac {
 	void WriteControl(const io_port_t, const io_val_t value, const io_width_t);
 
 	// The DSS is an LPT DAC with a 16-level FIFO running at 7kHz
-	static constexpr auto dss_7khz_rate_hz = 7000;
-	static constexpr uint8_t max_fifo_size = 16;
+	static constexpr auto SampleRateHz  = 7000;
```

Comment:
Make this `DisneySampleRate` or something?  This could clash with another similar looking constant and get confusing.

Reply:
That's better. I had an uneasy feeling when I did this... but I repressed it 😎 🤘🏻